src/TrustedProxyServiceProvider.php

```diff
@@ -37,9 +37,12 @@ private function registerPublishing()
      */
     public function register()
     {
-        $this->mergeConfigFrom(
-            __DIR__.'/../config/laravelcloudflare.php', 'laravelcloudflare'
-        );
+        if (! app()->configurationIsCached()) {
```

Comment:
@zhanang19 Any though on the psalm error?

Reply:
> PossiblyUndefinedMethod: Method Illuminate\Contracts\Foundation\Application::configurationIsCached does not exist


Method `configurationIsCached()` available at `Illuminate\Contracts\Foundation\CachesConfiguration` interface which is implemented at `Illuminate\Foundation\Application`.

But, since the `app()` helper is resolved as `Illuminate\Contracts\Foundation\Application`, psalm think that method is possibly undefined.